tally_ho/apps/tally/views/tally_manager.py

```diff
@@ -381,3 +389,66 @@ def post(self, request, *args, **kwargs):
             'status': 'OK',
             'elements_processed': elements_processed}),
                             content_type='application/json')
+
+
+class SetUserTimeOutView(LoginRequiredMixin,
+                         mixins.GroupRequiredMixin,
+                         mixins.ReverseSuccessURLMixin,
+                         SuccessMessageMixin,
+                         UpdateView):
+    model = SiteInfo
+    form_class = SiteInfoForm
+    group_required = groups.TALLY_MANAGER
+    template_name = "tally_manager/set_user_timeout.html"
+    success_url = 'tally-manager'
+
+    def get_object(self):
+        site_id = self.kwargs.get('site_id', None)
+
+        return get_object_or_404(Site, pk=site_id)
+
+    def get(self, *args, **kwargs):
+        user_idle_timeout = None
+        self.object = self.get_object()
+
+        try:
+            siteinfo = SiteInfo.objects.get(site__pk=self.object.pk)
+            user_idle_timeout = siteinfo.user_idle_timeout
+        except SiteInfo.DoesNotExist:
+            user_idle_timeout = getattr(settings, 'IDLE_TIMEOUT', 60)
+
+        form_class = self.get_form_class()
+        form = self.get_form(form_class)
+
+        return self.render_to_response(
+            self.get_context_data(form=form,
+                                  userIdleTimeout=user_idle_timeout))
+
+    def post(self, request, *args, **kwargs):
+        self.object = self.get_object()
+
+        form_class = self.get_form_class()
+        form = self.get_form(form_class)
+
+        if form.is_valid():
+            site_info = form.save()
+
+            if isinstance(site_info, SiteInfo):
+                self.success_message = _(
+                    u"Successfully set user timeout to %(user_idle_timeout)s"
+                    u" minutes"
+                    % {'user_idle_timeout': site_info.user_idle_timeout})
+
+            return redirect(self.success_url)
+
+        user_idle_timeout = None
+
+        try:
+            siteinfo = SiteInfo.objects.get(site__pk=self.object.pk)
+            user_idle_timeout = siteinfo.user_idle_timeout
+        except SiteInfo.DoesNotExist:
+            user_idle_timeout = getattr(settings, 'IDLE_TIMEOUT', 60)
```

Comment:
let's use the constant here, and also in the test files, no magic numbers please :)